app/src/main/java/com/babylon/wallet/android/presentation/settings/troubleshooting/importlegacywallet/ImportLegacyWalletViewModel.kt

```diff
@@ -439,14 +438,10 @@ class ImportLegacyWalletViewModel @Inject constructor(
 
     fun onContinueWithLedgerClick() {
         viewModelScope.launch {
-            val hasAtLeastOneLinkedConnector = getProfileUseCase().appPreferences.p2pLinks().isNotEmpty()
+            val hasAtLeastOneLinkedConnector = getProfileUseCase().appPreferences.p2pLinks.isNotEmpty()
 
             if (hasAtLeastOneLinkedConnector) {
                 useLedgerDelegate.onSendAddLedgerRequest()
-            } else if (hasAtLeastOneLinkedConnector.not()) {
-                _state.update {
-                    it.copy(shouldShowAddLinkConnectorScreen = true)
-                }
             } else {
                 _state.update {
                     it.copy(
```

Comment:
Please take a look at this. Else block would never be executed. So I had to change this method. Please make sure you read this and tell me if the change is ok.